modules/images/regenerate-existing-images/load.php

```diff
@@ -7,3 +7,79 @@
  * @since n.e.x.t
  * @package performance-lab
  */
+
+/**
+ * Register the background job taxonomy.
```

Comment:
Nit-pick, but function descriptions should begin in a third-person verb.
```suggestion
 * Registers the background job taxonomy.
```